xml/System/UriBuilder.xml

```diff
@@ -1118,7 +1118,7 @@ In either case, the display string does not contain the <xref:System.UriBuilder.
         </remarks>
         <exception cref="T:System.UriFormatException">The <see cref="T:System.UriBuilder" /> instance has a bad password.
 
-Note: In <see href="/previous-versions/br230232(v=vs.110)">.NET for Windows Store apps</see> or the <see href="/dotnet/standard/cross-platform/cross-platform-development-with-the-portable-class-library">Portable Class Library</see>, catch the base class exception, <see cref="T:System.FormatException" />, instead.
+<div class="NOTE"><p>NOTE</p><p>In <see href="/previous-versions/br230232(v=vs.110)">.NET for Windows Store apps</see> or the <see href="/dotnet/standard/cross-platform/cross-platform-development-with-the-portable-class-library">Portable Class Library</see>, catch the base class exception, <see cref="T:System.FormatException" />, instead.</p></div>
```

Comment:
@carlossanlop Do you see any issues with having HTML tags like this in the /// source? For example, if we were to backport this to dotnet/runtime.